From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Bogdan Migunov <b.migunov@omprussia.ru>
Date: Thu, 15 Oct 2020 23:27:31 +0300
Subject: [PATCH] APN settings for Letai (Russia). Fixes JB#51615

Added APN settings and MCC/MNC identifiers for network operator "Letai".
---
 serviceproviders.xml | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/serviceproviders.xml b/serviceproviders.xml
index 456072cd7..769d51345 100644
--- a/serviceproviders.xml
+++ b/serviceproviders.xml
@@ -12795,6 +12795,23 @@ conceived.
 			</apn>
 		</gsm>
 	</provider>
+	<provider>
+		<name>Letai</name>
+		<name xml:lang="ru">Летай</name>
+		<gsm>
+			<network-id mcc="250" mnc="27"/>
+			<apn value="internet.letai.ru">
+				<usage type="internet"/>
+				<name>Letai Internet</name>
+			</apn>
+			<apn value="mms">
+				<usage type="mms"/>
+				<name>Letai MMS</name>
+				<mmsc>https://mmsc:8002</mmsc>
+				<mmsproxy>172.24.121.5:8080</mmsproxy>
+			</apn>
+		</gsm>
+	</provider>
 	<provider>
 		<name>Skylink</name>
 		<name xml:lang="ru">Скайлинк</name>
